sklearn/cluster/tests/test_birch.py

```diff
@@ -14,10 +14,9 @@
 from sklearn.linear_model import ElasticNet
 from sklearn.metrics import pairwise_distances_argmin, v_measure_score
 
-from sklearn.utils._testing import assert_almost_equal
-from sklearn.utils._testing import assert_array_equal
-from sklearn.utils._testing import assert_array_almost_equal
-from sklearn.utils._testing import assert_warns
+from sklearn.utils._testing import (
+    assert_almost_equal, assert_array_equal,
+    assert_array_almost_equal)
```

Comment:
I prefer to keep the expanded import style:

```python
from sklearn.utils._testing import assert_almost_equal
from sklearn.utils._testing import assert_array_equal
from sklearn.utils._testing import assert_array_almost_equal
```

it's less likely to generated conflicts when merging PRs that just add or delete one import line.